src/server/project.ts

```diff
@@ -1604,6 +1604,16 @@ namespace ts.server {
             return !!this.program && this.program.isSourceOfProjectReferenceRedirect(fileName);
         }
 
+        /*@internal*/
+        protected getGlobalPluginSearchPaths() {
+            // Search any globally-specified probe paths, then our peer node_modules
+            return [
+                ...this.projectService.pluginProbeLocations,
+                // ../../.. to walk from X/node_modules/typescript/lib/tsserver.js to X/node_modules/

```

Comment:
Aren't you walking up three directories here? But from "X/node_modules/typescript/lib/" to "X/node_modules" is only two directories up.

Reply:
combinePath makes it `X/node_modules/typescript/lib/tsserver.js/../../../` and comes out with ``X/node_modules/` 
Note this didnt change as part of this PR.